### ISSUE TITLE
Narrator announces the same Hex values for all colors under header Primary focus color and secondary focus color in Reveal focus.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#584]
**Describe the bug**

**Steps to reproduce the bug**

Pre-requisites:
- Turn on Narrator (Keyboard Shortcut: Ctrl+Win+Enter).
- Set Narrator to item mode.

1. Launch XAML Control Gallery app.
2. Activate "Reveal focus" control button from All Controls page.
3. Navigate to Primary focus color and observe the Narrator description for selected color.
4. Activate the  Primary focus color button, in the pop-up opened, change the slider position and select some different color and again observe the Narrator's description for color selected.
5. Repeat the same steps for Secondary focus color also.     

**Expected behavior**

​- Narrator should provide the color name along with the hexadecimal values of the color.
- Narrator should provide correct Hexadecimal values of different color selected.

**Actual behavior**

- The narrator doesn't announce the color name but only provides hexadecimal values of color.
- Also, Narrator announces same hexadecimal values for every color selected.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/97644262-16876f80-1a07-11eb-8660-ea49418179af.png)

**Version Info**

Gallery version (found on Settings page): 1.2.16.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/20651263)
